mlir/include/mlir/Dialect/ArmSVE/IR/ArmSVE.td

```diff
@@ -273,6 +273,34 @@ def UmmlaOp : ArmSVE_Op<"ummla",
     "$acc `,` $src1 `,` $src2 attr-dict `:` type($src1) `to` type($dst)";
 }
 
+def UsmmlaOp : ArmSVE_Op<"usmmla", [Pure,
+                                    AllTypesMatch<["src1", "src2"]>,
+                                    AllTypesMatch<["acc", "dst"]>]> {
```

Comment:
This indentation is inconsistent with the other ops, but the existing indentation feels a bit ad-hoc. I like yours much more.

Would you mind updating other definitions so that we do maintain consistency? Probably as a separate PR to keep the history clean. Updating this PR instead would also be fine with me.

Reply:
It's inconsistent only if others were consistent with each other.

I've updated some, but really with a lack of a policy (or automated process) it's rather futile to try to guess what one or another reviewer would like.